share/spice/twitch/channel_info/twitch_channel_info.css

```diff
@@ -0,0 +1,6 @@
+/* Push channel info to new line */
+.zci--twitch_channel_info .c-info__title__sub{
```

Comment:
The template should already put the subtitle on a new line when space allows -- if you have the developer tools open in your browser, because of the reduced height it pushes it onto a single line